### PR TITLE
Add some basic commands

### PR DESCRIPTION
**Pull request by LNJplus**

### CODE REVIEW

File: mods/default/commands.lua [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+})
+
+minetest.register_chatcommand("kill", {
+        params = "",
+        description = "Kills yourself",
+        func = function(name, param)
+                local player = minetest.get_player_by_name(name)
+		if not player then return end
+                if minetest.setting_getbool("enable_damage") == false then
+                        minetest.chat_send_player(name, "[X] Damage is disabled on this server.")
```
Comment (BlockMen) [inline mods/default/commands.lua]: This is inconsistent. Why is here [X] added to the message?

File: mods/default/commands.lua [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                else
+			player:set_hp(0)
+                        minetest.chat_send_player(name, "[X] You suicided.")
```
Comment (BlockMen) [inline mods/default/commands.lua]: same here

File: mods/default/commands.lua [hunk 3]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (BlockMen) [inline mods/default/commands.lua]: plz remove empty line

**Comment by LNJplus** [comment]
Is it ok now?

**Comment by BlockMen** [comment]
Im not sure about the clear inv, otherwise yes.

**Comment by LNJplus** [comment]
I can rename the command to clear_inventory or something else or delete it completly.

**Comment by BlockMen** [comment]
No need to hurry, leave it as is for now.

**Comment by BlockMen** [comment]
Concerning the ```clear_inventory``` command i would like to have a conformation system. Something like "Do you really want delete your whole inventory? Y/N" chatmessage prompting.

But to get this merged you can decide what you prefer. Changing the command as described above or remove the command to get the rest merged now.

**Comment by LNJplus** [comment]
At the moment I have no desire to do this, so I'll deleted the command. Maybe I'll do that later.

**Comment by BlockMen** [comment]
a1e5eda3184c7496bb62209b9f1139d4f3cd9d88

**Pull request closed without merging**